clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -104,6 +104,16 @@ export const TimeSlotSelect = () => {
     }
   }, [minDate, setDate])
 
+  useEffect(() => {
+    if (preloadBooking && preloadBooking.start && preloadBooking.end && preloadBooking.provider.id) {
```

Comment:
Is it possible for `preloadBooking.provider` to be undefined? If so, we could do `preloadBooking.provider?.id`

Reply:
In theory maybe yes, though I have no TS errors. In reality it will never be undefined because the `preloadBooking` is just encoded and passed all the way through from `onClick` in the scheduler through any auth flow and back. I'll update just to be defensive.